### PR TITLE
Cloud integration documentation update

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

Update the cloud integration documentation to:
* Clarify requirements, reference site certification as required steps, 
* Move to catch-all cloudkeeper by default, so removing references to it
* Remove references to legacy Check-in (pre Keycloak) as it's been deprecated
* Improve the description of GOCDB registration

What's missing:
* Update the accounting to not reference the appliance
* Update the cloud-info page to move to catch-all by default

---

**Related issue :**

### CODE REVIEW

**Comment by enolfc** [comment]
Will progress slowly with the different sections, but already as draft PR to be able to visualise and comment

File: content/en/providers/cloud-compute/_index.md [hunk 1]
```diff
@@ -5,44 +5,47 @@ weight: 30
...
-- the projects you may be involved in as cloud provider.
-
-- the user communities you want to support (AKA Virtual Organisations or VOs).
-  You can also support the 'long-tail of science' through the vo.access.egi.eu VO.
-
-- the technologies (Cloud Management Framework) you want to provide.
-
+This documentation covers how to join the EGI Cloud federation (also known as
+Federated Cloud or FedCloud) as a provider.  If you are interested in joining
+please first contact EGI Operations team at
```
Comment (gwarf) [inline content/en/providers/cloud-compute/_index.md:10]: Contact name and address to be aligned (support@ won't reach ops)

**Comment by enolfc** [comment]
Rebased on top of #628

**Comment by enolfc** [comment]
Ready for review, let's have improvements in coming PRs

**Review by CatalinCondurache — APPROVED** [review]
Validated.

**Review by CatalinCondurache — APPROVED** [review]
validated

**Review by CatalinCondurache — APPROVED** [review]
validated

**Review by CatalinCondurache — APPROVED** [review]
validated

**Review by CatalinCondurache — APPROVED** [review]
validated

**Review by CatalinCondurache — APPROVED** [review]
validated

**Review by CatalinCondurache — APPROVED** [review]
LGTM

**Pull request merged**